gdpr/vendorlist-fetching_test.go

```diff
@@ -184,7 +184,7 @@ func TestDefaultVendorListPassthrough(t *testing.T) {
 			purposes: []int{2},
 		},
 	})
-	server := httptest.NewServer(http.HandlerFunc(mockServer(2, map[int]string{
```

Comment:
Both lists were originally preloaded and always pulled from cache. If we change the latest version from 2 to 1, it no longer preloads version 2 and attempts a lazy load. This exposed the bug.